File: 프로그래머스/lv2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.js
function solution(s) {
    var answer = "";
    s.split(" ").forEach(
        (e) =>
            (answer +=
                e.charAt(0).toUpperCase() +
                e.slice(1, e.length).toLowerCase() +
                " ")
    );
    return answer.slice(0, answer.length - 1);
}

File: 프로그래머스/lv2/12951. JadenCase 문자열 만들기/JadenCase 문자열 만들기.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string s) {
    string answer;
    
	for (int i = 0; i < s.length(); ++i)
	{
		if (i == 0 || isspace(s[i - 1]) && !isdigit(s[i]))
			answer += toupper(s[i]);
		else
			answer += tolower(s[i]);
	}

	return answer;
}